### PR TITLE
{math}[foss/2018a] DOLFIN v2018.1.0.post1, FIAT v2018.1.0, UFL v2018.1.0, ...

### PR DESCRIPTION
**Pull request by boegel**
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1521~~ + ~~#6844~~ (Boost.Python, Trilinos), ~~#6872~~ (pybind11), ~~#6874~~ (PETSc, SLEPc, petsc4py, slepc4py, Hypre), ~~#6875~~ (CGAL), ~~#6876~~ (PLY)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2041.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1748ad1a4461e7ebbd4a929c2b4629d7 for a full test report.

**Comment by bartoldeman** [comment]
Reopening to trigger travis

File: easybuild/easyconfigs/u/UFL/UFL-2018.1.0-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ for declaration of finite element discretizations of variational forms. 
+ More precisely, it defines a flexible interface for choosing finite element 
+ spaces and defining expressions for weak forms in a notation close to 
+ mathematical notation."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://bitbucket.org/fenics-project/ufl/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (bartoldeman) [inline easybuild/easyconfigs/u/UFL/UFL-2018.1.0-foss-2018a-Python-3.6.4.eb]: missing checksum

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 15 out of 17 (4 easyconfigs in this PR)
aw-4r13-n22 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b9d4898192864ae3f3ff9c3605b5d3dd for a full test report.

**Comment by boegel** [comment]
Hmm, looks like a missing library?

```
error: undefined reference to 'clock_gettime'
```

**Comment by bartoldeman** [comment]
> Hmm, looks like a missing library?
> 
> ```
> error: undefined reference to 'clock_gettime'
> ```

that's `-lrt`, but it's in Trilinos, not DOLFIN itself.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (4 easyconfigs in this PR)
aw-4r13-n22 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/138d5d45f0e09d14f6e7e9d1bd4cf9f4 for a full test report.

**Comment by bartoldeman** [comment]
Trilinos builds for me with the modified easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/1525. DOLFIN itself has no trouble.

**Pull request merged**